packages/client/package.json

```diff
@@ -47,5 +47,9 @@
   "homepage": "https://github.com/open-feature/js-sdk#readme",
   "devDependencies": {
     "@openfeature/shared": "0.0.2"
+  },
+  "typedoc": {
+    "displayName": "OpenFeature Web SDK",
+    "entryPoint": "./src/index.ts"
```

Comment:
This overrides the default entry point, which comes automatically from the package.json `main` field, and causes issues due to requiring source maps to get back to the TS..